### PR TITLE
Adds github configuration files for templates

### PR DESCRIPTION
**Pull request by ianphil**
Adds templates for:
 1. Issue stories
 2. Bug Issues
 3. Pull Requests

Github Action disabled by renaming to ".txt"... We can reuse later if someone wants to target the correct directory and add needed secrets.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## All Submissions:
+-------------------------------------
+* [YES/NO] Have you followed the guidelines in our Contributing [document](./CONTRIBUTING.md)?
+* [YES/NO] Have you added an explanation of what your changes do and why you'd like us to include them?
+* [YES/NO] I have updated the documentation accordingly.
+* [YES/NO/NA] I have added tests to cover my changes.
+* [YES/NO/NA] All new and existing tests passed.
+* [YES/NO/NA] My code follows the code style of this project.
+* [YES/NO/NA] I ran lint checks locally prior to submission.
+* [YES/NO] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
```
Comment (davidkpiano) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
* [ ] (Required) Have you followed the guidelines in our Contributing [document](./CONTRIBUTING.md)?
* [ ] (Required) Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes. 
* [ ] All new and existing tests passed.
* [ ] My code follows the code style of this project.
* [ ] I ran lint checks locally prior to submission.
* [ ] (Required) Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners,
+# the last matching pattern has the most precendence.
+
+# Core team members from orion
```
Comment (nmiodice) [inline .github/CODEOWNERS:5]:
```suggestion
# Core team members
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Push your branch to `origin` and submit a new pull request.
+
+You should expect code reviewers to comment on pull requests within one business day. We may suggest some changes or improvements or alternatives.
+
+Following an approval, the pull request submitter is responsible for squashing their commit messages completing their pr and deleting the branch.
+
+Some things that will increase the chance that your pull request is accepted:
+
+* Review our code review [guidance](https://github.com/Microsoft/code-with-engineering-playbook/blob/master/Engineering/CodeReviews.md).
```
Comment (nmiodice) [inline .github/CONTRIBUTING.md:17]: This link is broken

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+<!--- Verify first that your issue is not already reported on GitHub -->
+<!--- Also test if the latest release and develop branch are affected too -->
+
+##### SUMMARY
+<!--- Explain the problem briefly below -->
+
+##### COMPONENT NAME
+<!--- Write the name of the command, doc, task or feature below, use your best guess if unsure -->
+
+##### ORION VERSION
```
Comment (nmiodice) [inline .github/ISSUE_TEMPLATE/bug-report.md:14]:
```suggestion
#### VERSION
```

**Pull request merged**